docs/gettingstarted/getting-started-runtime-configuration.md

```diff
@@ -51,19 +51,32 @@ net.ipv4.ip_forward=1
 net.ipv4.ip_forward=1
 ```
 ## Select a Container Runtime and SC4S Configuration
+If no additional operating system requirements are specified, launching the selected configuration should be independent of the operating system as long as the containerization engine is installed correctly.
```

Comment:
If no additional operating system requirements are specified and the containerization engine is installed correctly, the selected configuration will launch independent of the operating system.